electrum/terms_and_conditions/en_UK.html

```diff
@@ -0,0 +1,10 @@
culpa aliquid, dicta provident! Laborum voluptatibus repellendus, architecto maiores eos obcaecati qui beatae officia doloremque quis eaque. Tempora voluptas ipsa veniam dignissimos aut natus sed, magnam corrupti? Quidem, nihil accusantium. Ipsum rem deleniti ea illum unde veniam officia ratione vitae consequatur, odit perspiciatis facere nisi ipsa alias sunt impedit culpa perferendis illo tenetur aperiam ullam quaerat! Similique qui iusto ipsa dolores voluptates voluptatem, quam, tenetur sint veniam, cupiditate id ut expedita perferendis corrupti iste ullam minus amet eum! Voluptatum, est aut ipsa ab dolor nisi, quisquam expedita quasi vitae architecto quia soluta repellendus. Numquam tenetur molestias enim nisi id distinctio porro facilis, illum veritatis. Beatae, placeat harum. Dignissimos optio quasi blanditiis sint possimus ducimus odit consequatur laborum iste tenetur modi perferendis corporis accusamus, repudiandae sunt? Similique, unde ipsum! Labore praesentium rem voluptate distinctio dolorum quidem reiciendis, quod facere iste doloribus similique ea tempore aperiam commodi debitis veritatis obcaecati iure voluptas, recusandae consequuntur consectetur ab veniam! Corrupti rem atque inventore doloribus voluptatem provident voluptate laborum aliquid ullam, autem distinctio repudiandae. Excepturi sed omnis distinctio, tenetur, at esse qui laborum quos ullam itaque nemo rerum corrupti earum repudiandae, odio commodi illo. Voluptatem earum, a tempora veniam error nisi nemo. Numquam, iusto? Mollitia voluptatem aspernatur corrupti repellat velit eaque exercitationem nostrum, beatae provident modi. Earum autem illum fugit, adipisci debitis, sed facilis, suscipit voluptas quaerat voluptatum expedita repudiandae velit itaque quo perferendis! Cumque, facilis veniam repellat quia fugiat minus eum! Iusto eaque sapiente laboriosam aperiam itaque mollitia sed, reiciendis et odit officiis eius necessitatibus ea numquam minima ut voluptatum praesentium consectetur dolorum explicabo omnis optio debitis. Officia quos aspernatur quae ducimus minus reprehenderit corporis inventore rerum, temporibus culpa aperiam praesentium suscipit quam autem assumenda reiciendis, odio voluptate sapiente, beatae dolorem! Suscipit nam pariatur unde similique fuga enim ducimus, modi harum animi alias quam sint, repellendus labore, ullam eaque corporis veritatis soluta illo! Eius explicabo rem mollitia quibusdam eos molestiae eveniet alias eum hic porro sint distinctio, facilis ipsam doloremque eligendi doloribus blanditiis deserunt dignissimos sed, totam maxime praesentium iusto. Quasi aliquam nemo quo expedita, obcaecati deleniti vel minima, earum soluta eligendi eveniet debitis iste nulla excepturi, consequatur commodi! Iure, beatae temporibus magni error cupiditate facilis asperiores. Odio suscipit, excepturi similique distinctio, consectetur cumque obcaecati natus consequuntur at explicabo quod dolores recusandae voluptatum hic magni repellat asperiores rerum quibusdam in!
```

Comment:
As I see it, we need to wait for proper content here

Reply:
done